rdbi-parent/pom.xml

```diff
@@ -42,7 +42,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <jackson.version>2.9.8</jackson.version>
+        <jackson.version>[2.9.10.1,)</jackson.version>
```

Comment:
why does rdbi need jackson?

Reply:
objects should be opaque no?

Reply:
there is a cache impl that "does it for you".
long term we can tear it out, but I just wanted to resolve the security alert for today.

Reply:
is jackson set up as bring your own?

Reply:
i'm not sure of a way to express that in _this_ maven pom, but in practice our usages of RDBI exclude jackson as a transient dep and specify the version that "we" want.